lib/request-processor.js

```diff
@@ -1,11 +1,30 @@
 const request = require('request')
-
+const os = require('os')
+const uuidv4 = require('uuid/v4')
+const logger = require('./logger')
+const assert = require('assert')
 module.exports = class RequestProcessor {
   constructor ({requestImpl}) {
     this._request = requestImpl || request
+    this._logger = logger
+  }
+
+  _log ({extensionUri, accessToken, resources, type}) {
+    this._logger.log(JSON.stringify({
+      'id': uuidv4(),
+      'type': type,
+      'user': accessToken.substr(7, 16), // get the first 16 characters of the user's access token
```

Comment:
this seems oddly specific and arbitrary?

Reply:
The full accessToken is very long and clutters the log. I find these first 16 characters would be good enough to determine which authenticated user executed the chaos event.

Thinking back to why this prop was added in the first place: The audit should give some identifier as to _who_ executed the event. If this is a person or some automated system; everything requires some accessToken in order to emit chaos.

Let me know your thoughts on what a good resolution would be here.